content/using_bookwyrm/privacy-controls.md

```diff
@@ -0,0 +1,165 @@
+---
+Title: Privacy Controls
+Date: 2025-05-26
+Order: 2
+---
+
+Bookwyrm has different privacy levels which let users control how public something is and who it is visible to.
+Do you want to share your reading with the internet, just your friends, or do you want to keep it private?
+
+There are four privacy levels: Public, Unlisted, Followers, and Private.
+Broadly speaking, Public is visible to anyone; Unlisted hides it from discovery pages; Followers is only visible to people who follow you; and Private is only visible to you.
+There are some slight differences in how these apply to different things in Bookwyrm.
+
+Throughout the website, you can check what something's privacy level is by the icon next to it.
+Public is a globe, Unlisted is an open lock, Followers is a closed lock, and Private is an envelope.
+
+Note: Anyone can just follow you and then be able to see all things you marked Followers-only.
+To limit this, go to `Settings - Edit Profile - Privacy` and enable 'Manually approve followers'.
+This allows you to vet follow requests, or limit them just to your friends.
+
+Other privacy settings are explained at the bottom of this page. [Click here](#privacy-related-settings) to scroll down to them.
```

Comment:
Linking to more descriptive text than "click here" is preferred for accessibility. Something like this would be better:

```md
Other privacy settings are explained [at the bottom of this page](#privacy-related-settings).
```